uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/contact-actions-menu.js

```diff
@@ -0,0 +1,40 @@
+var Promise = require('es6-promise').Promise;
+var page = require('./page');
+
+module.exports = (function () {
+  var contactActionsMenuButton = '#crm-contact-actions-link';
```

Comment:
I would prefer to use `contactActionsMenuButtonSelector` and `deleteUserAccountButtonSelector` instead. It is a bit longer, but clearly tells you what it is.